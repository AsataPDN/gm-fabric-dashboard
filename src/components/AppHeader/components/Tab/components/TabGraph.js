import styled from "styled-components";

import {
  contrastColor,
  spacingScale
} from "../../../../../style/styleFunctions";
import { COLOR_ALT_BACKGROUND } from "../../../../../style/styleVariables";

const COLOR_TAB_BACKGROUND_BASE = contrastColor(COLOR_ALT_BACKGROUND, 0.175);

const TabGraph = styled.div`
  color: ${contrastColor(COLOR_TAB_BACKGROUND_BASE, 0.8).string()};
  margin: 0 ${spacingScale(0.25)};
  height: ${spacingScale(3)};
`;

export default TabGraph;
