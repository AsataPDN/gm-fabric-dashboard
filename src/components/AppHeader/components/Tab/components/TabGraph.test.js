import React from "react";
import { shallow } from "enzyme";
import TabGraph from "./TabGraph";

describe("TabGraph", () => {
  it("matches snapshot", () => {
    const aTabGraph = shallow(<TabGraph />);
    expect(aTabGraph).toMatchSnapshot();
  });
});
