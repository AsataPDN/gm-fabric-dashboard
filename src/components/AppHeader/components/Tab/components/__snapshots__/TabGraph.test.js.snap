// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabGraph matches snapshot 1`] = `
.c0 {
  height: 24px;
  color: rgb(213,213,213);
  margin: 0 2px;
  padding-top: 4px;
  padding-bottom: 4px;
}

<div
  className="c0"
/>
`;
